Makefile

```diff
@@ -324,6 +329,9 @@ $(info Cargo features:)
 $(info   * Compilers: `$(bold)$(green)${compiler_features}$(reset)`.)
 $(info Rust version: $(bold)$(green)$(shell rustc --version)$(reset).)
 $(info NodeJS version: $(bold)$(green)$(shell node --version)$(reset).)
+ifeq ($(ENABLE_LLVM), 1)
+        $(info LLVM version: $(bold)$(green)$(shell llvm-config --version)$(reset).)
```

Comment:
The command available might be `llvm-config-12` or other one. It might be better to create a `LLVM_VERSION` var and just use it here?

Reply:
Well, the first choice is using generic `llvm-config`, and if now found, using version specific ones. In that case, the LLVM_VERSION variable is not set, but that can be fixed easily, to be used later.
I'll make the change so the shown version is more coherent with the actually selected one.

Reply:
Sounds good!